exersices/Ex02/bubble_sort.py

```diff
@@ -0,0 +1,24 @@
+def bubble_sort(data):
+    data = list(data)
+    length = len(data)
+    while length >= 1:
```

Comment:
Works. But causes confusion when walk-through. Not really bubble sort. Redundant comparison operation on sorted data.

Please look at the suggested solution.